modules/storage/storage-account/README.md

```diff
@@ -17,15 +17,46 @@ The output of the module is the ID of the created or existing Storage Account, w
+| `destKind`                        | `string` | Yes      | Indicates the type of storage account.                                                                                               |
+| `destSupportHttpsTrafficOnly`     | `bool`   | No       | Allows https traffic only to storage service if sets to true.                                                                        |
+| `destAllowBlobPublicAccess`       | `bool`   | No       | Allow or disallow public access to all blobs or containers in the destination storage account.                                       |
+| `destAllowCrossTenantReplication` | `bool`   | No       | Replication of objects between AAD tenants is allowed or not. For this property, the default interpretation is true.                 |
+| `destPublicNetworkAccess`         | `string` | No       | Allow or disallow public network access to Storage Account. Value is optional but if passed in, must be Enabled or Disabled.         |
+| `destMinimumTlsVersion`           | `string` | No       | Set the minimum TLS version to be permitted on requests to storage. The default interpretation is TLS 1.0 for this property.         |
+| `destDaysAfterLastModification`   | `int`    | No       | It will be deleted after the given amount of days.                                                                                   |
+| `destBlobType`                    | `string` | No       | Specifies the type of blob to manage the lifecycle policy.                                                                           |
+| `destChangeFeedEnabled`           | `bool`   | No       | Indicates whether change feed event logging is enabled for the Blob service.                                                         |
+| `destVersioningEnabled`           | `bool`   | No       | Versioning is enabled if set to true. To the destination storage account, set true.                                                  |
+| `destSkuName`                     | `string` | Yes      | The SKU name required for account creation; optional for update.                                                                     |
+| `ruleId`                          | `string` | No       | Rule Id is auto-generated for each new rule on destination account. It is required for put policy on source account.                 |
+| `policyId`                        | `string` | No       | This is the name to provide for objectReplicationPolicies.                                                                           |
+| `objectReplicationPolicy`         | `bool`   | No       | When performing object replication, it must be true and all resources necessary for the destination storage account will be created. |
+| `managedIdentityName`             | `string` | Yes      | User defined name to provide userAssignedIdentities resource.                                                                        |
```

Comment:
Can you expand on why managedIdentity params are required? Can these be optional?

Reply:
Yes, this can be optional and added default value to this param and also updated in readme.md file.